docs/getting-started/getting-started-darwin.md

```diff
@@ -181,10 +181,16 @@ Compile and install the rocksdb library
 curl -L https://github.com/facebook/rocksdb/pull/7714.patch | git apply -
  
 CFLAGS="-I$(brew --prefix)/include" LDFLAGS="-L$(brew --prefix)/lib" make shared_lib
+
```

Comment:
Can you remove the new line here to keep the code looking uniform (see other `if the above` comment below).